### PR TITLE
Prepare the morello elf document for the next release.

### PR DESCRIPTION
**Pull request by sbaranga-arm**

### CODE REVIEW

File: aaelf64-morello/aaelf64-morello.rst [hunk 1]
```diff
@@ -1,13 +1,13 @@
 ..
-   Copyright (c) 2020, Arm Limited and its affiliates.  All rights reserved.
```
Comment (stuij) [inline aaelf64-morello/aaelf64-morello.rst:2]: when there are only two years, it should be `2020, 2021`

File: aaelf64-morello/aaelf64-morello.rst [hunk 2]
```diff
@@ -1,13 +1,13 @@
...
+   Copyright (c) 2020-2021, Arm Limited and its affiliates.  All rights reserved.
    CC-BY-SA-4.0 AND Apache-Patent-License
    See LICENSE file for details
 
-.. |release| replace:: 2020Q4
-.. |date-of-issue| replace:: 21\ :sup:`st` December 2020
-.. |copyright-date| replace:: 2020
-.. |footer| replace:: Copyright © |copyright-date|, Arm Limited and its
-                      affiliates. All rights reserved.
```
Comment (stuij) [inline aaelf64-morello/aaelf64-morello.rst:10]: Now `Non-Confidential` kind of trails behind, without being enclosed by a period. Perhaps `All rights reserved, Non-Confidential`?

File: aaelf64-morello/aaelf64-morello.rst (hunk 1 above)
Comment (sallyarmneale) [reply aaelf64-morello/aaelf64-morello.rst:2]: Correct.

File: aaelf64-morello/aaelf64-morello.rst (hunk 2 above)
Comment (sallyarmneale) [reply aaelf64-morello/aaelf64-morello.rst:10]: No.  There must be a period after All Rights Reserved.
`All Rights Reserved. Non-Confidential.`

**Comment by sallyarmneale** [comment]
LGTM

**Comment by stuij** [comment]
Sorry @sallyarmneale, but pull requests need to be merged instead of closed, as this will apply the patch to the code-base.

**Pull request merged**